documents/description.md

```diff
@@ -2,8 +2,105 @@
 
 ## Motivation
 
+###### Why is this project useful or interesting, what problem are you trying to address, and why is a DSL an appropriate solution?
+This project is useful since it will attempt to be a tool for teaching novices programming while also challenging experienced 
+programmers in a fun game enviornment. I originally became interested in programming in 4th grade by learning how to script using
+_SCM Draft 2_ and _StarForge_, two 3rd party map editors made for creating custom scenarios for the popular computer game _Starcraft_.
+My goal is to create a similar more defined experience to help inspire new or novice programmers to learn programming while also 
+challenging experienced programmers to tackle some interesting AI and navigation problems.
+
+A DSL is an appropriate solution for this project since I want to embed the language inside of a game. Although I definetly want this
+language to feel C-like, I want to include very detailed feedback/errorchecking to the user, especially useful for new programmers,
```

Comment:
By "C-like", do you mean syntactically, semantically (e.g., with memory management), or both (or neither)? And how does that answer relate to the goal of being accessible to novice programmers?